lib/dependencies/ContextDependencyHelpers.js

```diff
@@ -5,19 +5,43 @@
 var ContextDependencyHelpers = exports;
 
 ContextDependencyHelpers.create = function(Dep, range, param, expr, options) {
```

Comment:
ContextDependencyHelpers is not only used by `System.import` but also for CommonJs and AMD.

So please add also test cases for 

* ``require(`./file`)``
* ``require(`./folder/${expr}`)``
* ``require([`./file`])``
* ``require([`./folder/${expr}`])``
* ``require(`./prefix/${expr1}/middle/${expr2}.postfix`)``